superset/stackable/superset_config.py

```diff
@@ -1,4 +1,6 @@
 import os
+from superset.stats_logger import StatsdStatsLogger
 
 SECRET_KEY = os.environ.get("SECRET_KEY")
 SQLALCHEMY_DATABASE_URI = os.environ.get("SQLALCHEMY_DATABASE_URI")
+STATS_LOGGER = StatsdStatsLogger(host='0.0.0.0', port=9125)
```

Comment:
Would it make sense to set the metrics port as an env variable from the operator?